src/Data.hs

```diff
@@ -39,13 +40,15 @@ data CaptureData = CaptureData
 
 data WindowData = WindowData
         { wActive :: Bool
+        , wHidden :: Bool
         , wTitle :: Text
         , wProgram :: Text
+        , wDesktop :: Text
         }
   deriving (Show, Read, Generic, NFData)
 
 instance Default WindowData where
- def = WindowData False "" ""
+ def = WindowData False False "" "" ""
```

Comment:
I wonder if `hidden = True` might not be a better default for old data? Not sure, though.

Reply:
For the rules that I'm going to write using this new feature, it very well might be, yeah. Perhaps `hidden = not active` would be even better. It's still wrong, but at least it won't confuse rules that take into account visible windows.

Reply:
👍🏻 to `hidden = not active`